java-checks/src/test/files/checks/security/HostnameVerifierImplementationCheck.java

```diff
@@ -0,0 +1,72 @@
+import javax.net.ssl.HostnameVerifier;
```

Comment:
This one is an interface, I would add a testcase where this is implemented nicely and wrongly with a class definition. 

```
class A implements HostnameVerifier {
// wrong implemt
}
class B implements HostnameVerifier {
// correct implem
}